jwt/api.py

```diff
@@ -16,7 +16,7 @@
 
 
 class PyJWT(object):
-    def __init__(self, algorithms=None):
+    def __init__(self, algorithms=None, options=None):
```

Comment:
what do you think about the keyword being, `verification_options`, so they are well categorized.